### PR TITLE
Add bedrock multimodal build-in function usage example in doc

### PR DESCRIPTION
**Pull request by zane-neo**
### Description
Add bedrock multimodal build-in function usage example in doc

### Related Issues
https://github.com/opensearch-project/ml-commons/issues/3060

### Check List
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] API changes companion pull request [created](https://github.com/opensearch-project/opensearch-api-specification/blob/main/DEVELOPER_GUIDE.md).
- [ ] Commits are signed per the DCO using `--signoff`.
- [ ] Public documentation issue/PR [created](https://github.com/opensearch-project/documentation-website/issues/new/choose).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/ml-commons/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md [hunk 1]
```diff
@@ -46,6 +47,21 @@ POST /_plugins/_ml/connectors/_create
         "content-type": "application/json",
         "x-amz-content-sha256": "required"
       },
+      "request_body": "{\"body\":{\"inputText\": \"${parameters.inputText}\", \"inputImage\": \"${parameters.inputImage}\"}}",
+      "pre_process_function": "connector.pre_process.bedrock.multimodal_embedding",
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:51]: Explain from which version user can use this ?

**Review by xinyual — APPROVED** [review]

File: docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md [hunk 2]
```diff
@@ -82,6 +103,21 @@ POST /_plugins/_ml/connectors/_create
         "content-type": "application/json",
         "x-amz-content-sha256": "required"
       },
+      "request_body": "{\"body\":{\"inputText\": \"${parameters.inputText}\", \"inputImage\": \"${parameters.inputImage}\"}}",
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:106]: This is not correct. Copy line 67 here 
```
"request_body": "{ \"inputText\": \"${parameters.inputText:-null}\", \"inputImage\": \"${parameters.inputImage:-null}\" }",
```

Comment (zane-neo) [reply docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md]: Fixed and tested, not seeing issue now.

**Review by model-collapse — APPROVED** [review]

File: docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md [hunk 3]
```diff
@@ -46,6 +49,21 @@ POST /_plugins/_ml/connectors/_create
...
+      "post_process_function": "connector.post_process.bedrock.embedding"
+    }
+  ]
+}
+```
+**Use painless script**
+```json
+POST /_plugins/_ml/connectors/_create
+{
+  ... //same above
```
Comment (ylwu-amzn) [inline docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:63]: How about just show the complete request ?

Comment (zane-neo) [reply docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:63]: I think for most cases if users are using the default process function, this simplification can save time to scroll down to copy following parts.

File: docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md [hunk 4]
```diff
@@ -46,6 +49,21 @@ POST /_plugins/_ml/connectors/_create
...
         "x-amz-content-sha256": "required"
       },
+      "request_body": "{\"inputText\": \"${parameters.inputText:-null}\", \"inputImage\": \"${parameters.inputImage:-null}\"}",
+      "pre_process_function": "connector.pre_process.bedrock.multimodal_embedding",
+      "post_process_function": "connector.post_process.bedrock.embedding"
+    }
+  ]
+}
+```
+**Use painless script**
```
Comment (dhrubo-os) [inline docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:59]: If building function is way more straightforward than painless script, why they should use this? What's the benefit? If there's any benefit of using painless script we should explain here.

Comment (zane-neo) [reply docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:59]: User might using an older version OpenSearch, so we need to keep this for those versions.

Comment (dhrubo-os) [reply docs/remote_inference_blueprints/bedrock_connector_titan_multimodal_embedding_blueprint.md:59]: Can we add that information in the blueprint? [If your OS version is 2.15 or older, you need to use painless script]

**Review by ylwu-amzn — APPROVED** [review]

**Pull request merged**